src/dlgTriggerEditor.cpp

```diff
@@ -5157,7 +5157,7 @@ void dlgTriggerEditor::fillout_form()
                 }
                 else
                 {
-                    if( pT->shouldBeActive() )
+                    if( timer->shouldBeActive() )
```

Comment:
The nesting of the `if(...)`s are not right here, the `else if( timer->shouldBeActive()` above here is wrong and throws off the block layout, change it (back ?) to an `else` so this line is the first inside the else - i.e. remove diff "after" lines 5154 to 5157 inclusive.